packages/drafts-realm/task-form.gts

```diff
@@ -0,0 +1,315 @@
+  @tracked accountPlaceholder = this.args.model.account || 'Select';
+
+  get getAccountsNames() {
+    let allAccounts = this.args.model.accounts || [];
+    return allAccounts.map((o) => ({ name: o.accountName }));
+  }
+
+  @action updateAccount(type: { name: string }) {
+    this.selectedAccount = type;
+    this.args.model.account = type.name;
+  }
+
+  <template>
+    <CardContainer @displayBoundaries={{false}} class='card-container'>
+
+      <BoxelSelect
+        @placeholder={{this.accountPlaceholder}}
+        @selected={{this.selectedAccount}}
+        @onChange={{this.updateAccount}}
+        @options={{this.getAccountsNames}}
+        class='select'
+        as |item|
+      >
+        <div>{{item.name}}</div>
+      </BoxelSelect>
+
+    </CardContainer>
+
+    <style>
+      .select {
+        padding: var(--boxel-sp-xs);
+        background-color: white;
+      }
+    </style>
+  </template>
+}
+
+class EmbeddedSecForAccountField extends Component<typeof AccountField> {
+  <template>
+    <CardContainer @displayBoundaries={{false}} class='card-container'>
+      {{this.args.model.account}}
+    </CardContainer>
+
+    <style>
+      .card-container {
+        background: transparent;
+      }
+    </style>
+  </template>
+}
+
+class AccountField extends FieldDef {
+  @field accounts = linksToMany(() => CrmAccount, {
+    description: `CRM Accounts`,
+  });
+  @field account = contains(StringField, {
+    description: `Account Name`,
+  });
+  static edit = EditSecForAccountField;
+  static embedded = EmbeddedSecForAccountField;
+}
+
+/* Subject */
+class EditSecForSubjectField extends Component<typeof SubjectField> {
+  @tracked selectedSubject = {
+    name: this.args.model.title || 'Select',
+  };
+
+  @tracked subjectPlaceholder = this.args.model.title || 'Select';
+
+  @tracked subjectOptions = [
+    { name: 'None' },
+    { name: 'Email' },
+    { name: 'Call' },
+    { name: 'Send Letter' },
+    { name: 'Send Quote' },
+    { name: 'Other' },
+  ] as Array<CategorySignature>;
+
+  @action updateSubject(type: { name: string }) {
+    this.selectedSubject = type;
+    this.args.model.title = type.name;
+  }
+
+  <template>
+    <CardContainer @displayBoundaries={{false}} class='card-container'>
+
+      <BoxelSelect
+        @searchEnabled={{true}}
+        @searchField='name'
+        @placeholder={{this.subjectPlaceholder}}
+        @selected={{this.selectedSubject}}
+        @onChange={{this.updateSubject}}
+        @options={{this.subjectOptions}}
+        class='select'
+        as |item|
+      >
+        <div>{{item.name}}</div>
+      </BoxelSelect>
+
+    </CardContainer>
+
+    <style>
+      .select {
+        padding: var(--boxel-sp-xs);
+        background-color: white;
+      }
+    </style>
+  </template>
+}
+
+class EmbeddedSecForSubjectField extends Component<typeof SubjectField> {
+  <template>
+    <div class='subject'>{{this.args.model.title}}</div>
+
+    <style>
+      .subject {
+        margin: 0px;
```

Comment:
Why is this 0. Can you use a CardContainer or GridContainer or something like this?

Reply:
Remind me the reason you don't use container